platforms/documentation/docs/src/snippets/bestPractices/avoidJdkCollectionApiOnFileCollections-do/kotlin/lib/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+import org.gradle.api.artifacts.Configuration.State.RESOLVED
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api("org.apache.commons:commons-lang3:3.12.0")
+}
+
+// tag::do-this[]
+tasks.register<Zip>("goodZippingTask") {
+    from(configurations.runtimeClasspath) // <1>
```

Comment:
I am not sure if we want to give this example. Gradle generally does not allow Configuration as an input to a task, so this is doing an implicit conversion to a `FileCollection`. Should we have a task that uses ConfigurableFileCollection and on that we call .from()